splight/client/client.go

```diff
@@ -10,12 +10,14 @@ import (
 	"time"
 
 	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/splightplatform/terraform-provider-splight/splight/settings"
 )
 
 // Client holds the configuration needed to communicate with a server
 type Client struct {
 	hostname   string       // Server hostname or IP address
 	authToken  string       // Authorization token for HTTP requests
+	orgID      string       // Splight organization ID
```

Comment:
Should either remove this or compute it on new client.
See if its usable for getting nodes